src/main/scala/intellij/haskell/module/HaskellModuleBuilder.scala

```diff
@@ -221,9 +223,12 @@ object HaskellModuleBuilder {
   private def downloadHaskellPackageSources(project: Project, stackPath: String, haskellPackages: Seq[HaskellPackageInfo], progressIndicator: ProgressIndicator) = {
     val step = 0.8 / haskellPackages.size
     var progressFraction = InitialProgressStep
-    haskellPackages.flatMap { packageInfo =>
-      val fullName = packageInfo.name + "-" + packageInfo.version
-      val stdErr = CommandLine.runProgram(Some(project), project.getBasePath + File.separator + LibName, stackPath, Seq("unpack", fullName), 10000, captureOutputToLog = true, logErrorAsInfo = true).map(_.getStderr)
+    haskellPackages.filter(packageInfo => {
```

Comment:
That is because some part of the haskellPackages are no longer need to be downloaded but we still need to get all the haskell package names.
>I do not understand this change, you filter and filter not on same condition..